pages/index.tsx

```diff
@@ -85,6 +86,35 @@ const AssetCardTitle = styled(Card.HeaderTitle)`
   overflow: hidden;
 `
 
+const Hero = styled.section`
```

Comment:
is there a reason you elected not to extend the `Section` styled component?

i feel like the search bar (which i believe we've decided to remove 🫣) is too close to the hero. `Section` has some nice margins that adds nice spacing between each section.

Reply:
Haha, no. I actually on purpose didn't touch the the spacing and stuff with the search bar since we talked about to remove it. Should we just remove it, and fix the spacing?

Reply:
i don't think we should remove the navbar. even temporarily. i guess we could fix the spacing once we've replaced navbar with the eds topbar :+1:

Reply:
Didn't talk about te navbar, just the search input field at the middle of the page